core/src/meta/methods.rs

```diff
@@ -2,16 +2,29 @@ use syn::*;
 
 use super::types::TypeName;
```

Comment:
Suggestion: `pub use` these types into *mod.rs* such that you don't need the extra level of namespace